tox.ini

```diff
@@ -66,3 +66,18 @@ deps={[testenv:optional]deps}
 envdir={[testenv:optional]envdir}
 recreate={[testenv:fast]recreate}
 setenv={[testenv:fast]setenv}
+
```

Comment:
Is there a good reason to add this to `tox.ini` instead of just adding to the `run_travis.sh` script?